integration/docker/entrypoint.sh

```diff
@@ -89,8 +90,10 @@ function mountAlluxioRootFSWithFuseOption {
   fi
 
   # Unmount first if cleanup failed and ignore error
-  ! integration/fuse/bin/alluxio-fuse unmount /alluxio-fuse
-  exec integration/fuse/bin/alluxio-fuse mount -n ${fuseOptions} /alluxio-fuse /
+  ! mkdir -p ${MOUNT_POINT}
+  ! umount ${MOUNT_POINT}
+  #! integration/fuse/bin/alluxio-fuse unmount ${MOUNT_POINT}
```

Comment:
as far as i remember, this command kills the fuse daemon and is also called in `preStop`.  is this still needed when the `preStop` hook is not executed for an unclean shutdown?

Reply:
It's needed for the case `Transport endpoint is not connected` when the fuse daemon is killed for OOM issue.  We've tested it in customer scenario.